frontend/src/pages/team/createProject.vue

```diff
@@ -186,26 +200,22 @@ export default {
 
         const templateList = await templatesApi.getTemplates()
         this.templates = templateList.templates.filter(template => template.active)
-
         this.init = true
 
-        setTimeout(() => {
-            // There must be a better Vue way of doing this, but I can't find it.
-            // Without the setTimeout, the select box doesn't update
-
+        this.$nextTick(() => {
             if (this.projectTypes.length === 0) {
                 this.errors.projectTypes = 'No project types available. Ask an Administrator to create a new project type'
             }
 
             if (!this.sourceProjectId) {
-                this.input.stack = this.stacks.length > 0 ? this.stacks[0].value : ''
-                this.input.template = this.templates.length > 0 ? this.templates[0].value : ''
+                this.input.stack = this.stacks.length > 0 ? this.stacks[0].id : ''
```

Comment:
Is this going to override the default stack if it's not the first one in the array?

Reply:
No - this code is marginally redundant. It will get called before the user has selected a ProjectType, so in fact this.stacks will always be empty. When they select a ProjectType it triggers the watcher (line 163) which loads the stacks for that type and selects the right one.